desktop/apps/desktop.games.solitaire.js

```diff
@@ -1,4 +1,5 @@
 desktop.games_solitaire = {};
+const intervals = [];
```

Comment:
We can't have apps exposing globally scoped variables unless absolutely required.

Could you rescope this into `desktop.game_solitaire` and perhaps remove the added `while` loop and rename the variables?

Thank you so much!